pages/dr3/issues.rst

```diff
@@ -7,65 +7,65 @@ DR3.1 Update to sweeps and Tractor Files
 ========================================
 Issue
 -----
-   During final processing checks on DR3 it was noticed that 36 bricks had been corrupted in a
-   read/write error. The corrupted files corresponding to these 36 bricks were::
+During final processing checks on DR3 it was noticed that 36 bricks had been corrupted in a
+read/write error. The corrupted files corresponding to these 36 bricks were::
 
-     tractor-1763p240.fits tractor-1821p295.fits tractor-1888p270.fits tractor-1916p295.fits
-     tractor-1934p255.fits tractor-1942p130.fits tractor-1942p255.fits tractor-1942p320.fits
-     tractor-1951p237.fits tractor-1896p300.fits tractor-1985p267.fits tractor-1986p237.fits
-     tractor-1986p272.fits tractor-1965p300.fits tractor-2030p270.fits tractor-2040p287.fits 
-     tractor-2049p237.fits tractor-2054p187.fits tractor-2063p300.fits tractor-2087p305.fits
-     tractor-2087p272.fits tractor-2109p317.fits tractor-2115p317.fits tractor-2116p302.fits
-     tractor-2117p307.fits tractor-2176p097.fits tractor-2204p252.fits tractor-2224p285.fits
-     tractor-2254p290.fits tractor-2261p257.fits tractor-2290p292.fits tractor-2295p320.fits
-     tractor-2310p320.fits tractor-2318p302.fits tractor-2334p200.fits tractor-2496p257.fits
+  tractor-1763p240.fits tractor-1821p295.fits tractor-1888p270.fits tractor-1916p295.fits
```

Comment:
Is there a particular reason to favor subheadings over dictionary lists here?  Also, by removing the indenting in this particular spot, you've actually undone the fixed-width formatting.